### ISSUE TITLE
Lobulated lesion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10082]
**Preferred term label:**
Lobulated lesion

**Synonyms**

**Definition (free text, please give PubMed ID)**
Often used to describe a mammography finding

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mucinous breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001048
- label: Lobulated
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality that displays ultiple indentations and/or elevations producing an irregular surface contour.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10082
- synonyms:
